### PR TITLE
Updating MS Learn learning paths to Innovate

### PR DESCRIPTION
**Pull request by scottburmester**
Added a link to a complementary MS Learn learning path which supports the Innovate section of CAF. Part of an exercise with Brian Blanchard to provide more specific guidance to our customers by recommending complementary learning paths from MS Learn.

### CODE REVIEW

**Comment by PRMerger7** [comment]
@scottburmester : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by jborsecnik** [comment]
Note: Must remove en-us after reviewed.

File: docs/innovate/index.md [hunk 1]
```diff
@@ -28,3 +28,13 @@ The cloud enables new forms of data within common platforms. Increasing velocity
...
+
+## Suggested Skills
+
+Microsoft Learn is a new approach to learning. Readiness for the new skills and responsibilities that come with cloud adoption doesn't come easily. Microsoft Learn provides a more rewarding approach to hands-on learning that helps you achieve your goals faster. Earn points and levels, and achieve more!
+Here are a couple of examples of role-specific learning paths on Microsoft Learn which align with the Innovate portion of the Cloud Adoption Framework
+
+[Administer containers in Azure](https://docs.microsoft.com/en-us/learn/paths/administer-containers-in-azure/): Azure Container Instances are the quickest and easiest way to run containers in Azure. This learning path will teach you how to create and manage your containers, and how ACI can be used to provide elastic scale for Kubernetes.
```
Comment (JanetCThomas) [inline docs/innovate/index.md]: Remove en-us/ from the URLs

File: docs/innovate/index.md [hunk 2]
```diff
@@ -28,3 +28,13 @@ The cloud enables new forms of data within common platforms. Increasing velocity
...
+
+[Create serverless applications](https://docs.microsoft.com/en-us/learn/paths/create-serverless-applications/):Azure Functions enable the creation of event driven, compute-on-demand systems that can be triggered by various external events. Learn how to leverage functions to execute server-side logic and build serverless architectures.
```
Comment (JanetCThomas) [inline docs/innovate/index.md]: remove en-us https://docs.microsoft.com/learn/paths/create-serverless-applications/
Add space after colon.   ...applications/): Azure Functions...

File: docs/innovate/index.md (hunk 1 above)
Comment (JanetCThomas) [reply docs/innovate/index.md]: change: Azure Container Instances
To:  Azure Container Instances (ACI)

**Comment by PRMerger14** [comment]
@scottburmester : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@BrianBlanchard : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
Resolved the merge conflicts.

**Pull request merged**